sdk/communication/AzureCommunicationChat/Source/Models/Message.swift

```diff
@@ -136,15 +148,24 @@ public struct Message: Codable {
         self.version = try container.decode(String.self, forKey: .version)
 
         // Convert ChatMessageContent to MessageContent
-        let content = try? container.decode(ChatMessageContent.self, forKey: .content)
-        self.content = MessageContent(from: content!)
+        let chatMessageContent = try? container.decode(ChatMessageContent.self, forKey: .content)
+
+        if let content = chatMessageContent {
+            self.content = try? MessageContent(from: content)
+        } else {
+            self.content = nil
+        }
```

Comment:
If you let `from` accept a nil value, you could handle this in the initializer and not need this if/else logic